drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c

```diff
@@ -685,22 +727,25 @@ static int bcmf_ifdown(FAR struct net_driver_s *dev)
 {
   FAR struct bcmf_dev_s *priv = (FAR struct bcmf_dev_s *)dev->d_private;
   irqstate_t flags;
+  int ret = OK;
```

Comment:
why do we need `ret`? `OK` only is returned anyway

Reply:
Done